docs/sources/mimir/release-notes/v2.10.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Grafana Mimir version 2.10 release notes"
+menuTitle: "V2.10 release notes"
+description: "Release notes for Grafana Mimir version 2.10"
+weight: 1000
+---
+
+# Grafana Mimir version 2.10 release notes
+
+Grafana Labs is excited to announce version 2.10 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For the complete list of changes, see the [changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Added support for rule filtering** by passing `file`, `ruler_group` and `rule_name` parameters to the ruler endpoint `/api/v1/rules`.
+- **Added support to only count series that are considered _active_** through the Cardinality API endpoint `/api/v1/cardinality/label_values` by passing the `count_method` parameter. You can set it to `active` to count only series that are considered _active_ according to the `-ingester.active-series-metrics-idle-timeout` flag setting rather than counting all in-memory series.
+- **Reduced the overall memory consumption** by changing the internal data structure for labels. Expect ingesters to use around 15% less memory with this change, depending on the pattern of labels used, number of tenants, etc.
+- **Reduced the memory usage of the Active Series Tracker** in the ingester.
+- **Added a buffered logging implementation** that can be enabled through the `-log.buffered` CLI flag. This should reduce contention and resource usage under heavy usage patterns.
+- **Improved the performance of the OTLP ingestion** and more detailed information was added to the traces in order to make troubleshooting problems easier.
+- **Improved the performance of series matching in the store-gateway** by always including the `__name__` posting group causing a reduction in the number of object storage API calls.
+- **Improved the performance of _label values with matchers_** calls when number of matched series is small. If you're using Grafana to query Grafana Mimir make sure you configure the right version in your Prometheus datasource configuration.
+- **Support to cache cardinality, label names and label values** query responses in query frontend. The cache will be used when `-query-frontend.cache-results` is enabled, and `-query-frontend.results-cache-ttl-for-cardinality-query` or `-query-frontend.results-cache-ttl-for-labels-query` set to a value greater than 0.
+- **Reduce wasted effort spent computing results that won't be used** by making queriers cancel the requests sent to a zone upon receiving first error from that zone.
```

Comment:
I'd clarify that this refers to ingesters

Reply:
Clarified, thanks.